### DISCUSSION TITLE
Documentation about parseBuildDiagnostics?

### DISCUSSION THREAD

**Discussion by juxeii** [microsoft/vscode-cmake-tools#2486] [Q&A] [answered]
Is there a documentation about `cmake.parseBuildDiagnostics`?
I want that the build output of cmake is parsed and that I can jump to the cpp file in case there is a compiler error. 
Is `parseBuildDiagnostics` the correct setting for this? I enabled it, but it does not provide jump links in case of compiler errors in the output window.

**Comment by bobbrow** [comment, marked as answer]
`cmake.parseBuildDiagnostics` is a setting that intended to allow you to turn off the red underlines (they should show up by default), but at some point before we took over maintainer duties for this extension the feature was removed.  We intended to make it work again, but it got buried in the backlog.  #1932

But if you're saying that you're not getting any errors showing up in the PROBLEMS window, then please open an issue and provide some more details about how we can reproduce the issue.  It's currently working for me in the projects I've tried.

**Comment by juxeii** [comment]
So, if I start a cmake build, and there are compile errors seen in the OUTPUT window(with file paths from the compiler), those can be tracked in the PROBLEMS window?
I thought it is possible to just CTRL+click the error file paths of the compiler in the OUTPUT window.
E.g. in a normal terminal, I can CTRL+click in case of errors, when building with Ninja.

Anyway, thx for explanation.

**Reply by bobbrow** [reply]
> So, if I start a cmake build, and there are compile errors seen in the OUTPUT window(with file paths from the compiler), those can be tracked in the PROBLEMS window?

Yes, those errors from the OUTPUT window should also appear in the PROBLEMS window.
